### ISSUE TITLE
Unable to sign in Kodi 19.1.0 | Platform Windows | Pcloud EU

### ISSUE THREAD

**Issue by shubham1358** [gdomenici/plugin.video.pcloud-video-streaming#38]
```
2021-08-23 02:42:27.789 T:16816   ERROR <general>: ERROR: cannot logon to pCloud: Error calling userinfo: Log in failed
2021-08-23 02:42:27.789 T:16816    INFO <general>: CPythonInvoker(3, C:\Users\Shubham\AppData\Roaming\Kodi\addons\plugin.video.pcloud-video-streaming\addon.py): script aborted
2021-08-23 02:42:27.798 T:16816    INFO <general>: Python interpreter stopped
2021-08-23 02:42:27.798 T:6676    ERROR <general>: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.pcloud-video-streaming/?content_type=video
2021-08-23 02:42:27.807 T:11972   ERROR <general>: CGUIMediaWindow::GetDirectory(plugin://plugin.video.pcloud-video-streaming/?content_type=video) failed
```

**Comment by gdomenici** [comment]
Hi @shubham1358 ,

This is unexpected. It seems to work for me on exactly the same configuration as you have.

Maybe if you please try this:

1. Close Kodi,
2. Delete the directory C:\Users\Shubham\AppData\Roaming\Kodi\userdata\addon_data\plugin.video.pcloud-video-streaming
3. Reopen Kodi, and go to the PCloud Video Streaming add-on

You will be asked to reauthenticate. In principle, it should work. If not, maybe double-check if the credentials are still valid on the PCloud website?

**Comment by gdomenici** [comment]
This should be fixed by [version 1.5.1](https://github.com/gdomenici/plugin.video.pcloud-video-streaming/releases/tag/1.5.1) - now added a setting to let one choose eapi.pcloud.com as an endpoint.
